CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.10.5
+  - Features Added
```

Comment:
0.10.5?

Reply:
that is one of the things the outstanding changelog PR fixes